test/e2e/features/pageObjects/mailinatorService.js

```diff
@@ -7,7 +7,7 @@ class MailinatorService{
         this.currentInboxLabel = "#inbox_page_title .title_left:nth-of-type(3) .top_search";
         this.latestEmailRowCssSelector = ".x_panel .x_content tbody tr:nth-of-type(1) td:nth-of-type(3)"
         this.activationEmaillinkCSsSelector = ".content tr:nth-of-type(2) td:nth-of-type(2) a:nth-of-type(1)";
-
+//a[contains(text(),'Confirm your email address')]
```

Comment:
Being finicky I'd say commented out code should be removed if no longer required.